packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -365,14 +365,13 @@ class FlexLayoutProperties extends LayoutProperties {
 
   static FlexLayoutProperties _buildNode(RemoteDiagnosticsNode node) {
     final Map<String, Object?> renderObjectJson = node.renderObject!.json;
-    final properties = renderObjectJson['properties'] as List<Object?>;
+    final properties = (renderObjectJson['properties'] as List<Object?>)
+        .cast<Map<dynamic, dynamic>>();
```

Comment:
Can this not be cast as a `Map<String, String>`?

Reply:
Mm, not quite, the objects are `Map<String, Object?>` (see [CI](https://github.com/flutter/devtools/actions/runs/7172643400/job/19530303729?pr=6931)). I'll try `String, Object?`.

Reply:
Ah yes, this one works; nice improvement.